packages/blinks-core/src/api/Action/Action.ts

```diff
@@ -333,6 +329,25 @@ export class Action {
     );
   }
 
+  static update(
```

Comment:
maybe it's better to just use a constructor instead of update method?

Reply:
naming of this method is misleading, since it doesn't update anything

Reply:
we don't expose `_data`, so I'd personally prefer to have a method to create a new instance

Reply:
@zhelezkov changed sematics a bit